master/internal/resourcemanagers/resource_manager_config.go

```diff
@@ -83,8 +123,9 @@ func DefaultRMConfig() *ResourceManagerConfig {
 // DefaultAgentRMConfig returns the default determined resource manager configuration.
 func DefaultAgentRMConfig() *AgentResourceManagerConfig {
 	return &AgentResourceManagerConfig{
-		SchedulingPolicy: "fair_share",
-		FittingPolicy:    "best",
+		Scheduler:              defaultSchedulerConfig(),
+		DefaultGPUResourcePool: defaultResourcePoolName,
```

Comment:
Right now we only use this `DefaultAgentRMConfig` for when the user specifies `scheduler` instead of a `resource_manager` in which case the behavior is unchanged from the previous implementation, do we ever expect to use this function for anything else?